lms/static/scripts/frontend_apps/components/ErrorDisplay.js

```diff
@@ -127,30 +106,17 @@ Details: ${formatErrorDetails(error) || 'N/A'}
  * @param {ErrorDisplayProps} props
  */
 export default function ErrorDisplay({ children, description = '', error }) {
-  // If `serverMessage` is extant on `error`, prefer it to `error.message` even
-  // if `serverMessage` is empty — In cases where we are displaying error
-  // information provided by the backend (i.e. `APIError`), we do not want
-  // to render the JS Error instance's `message` as it likely does not apply
-  const message = error.serverMessage ?? error.message ?? '';
-
-  // Create an error status message from the combination of `description` and
-  // `message`. As neither of these are guaranteed to be present, the
-  // resulting string may be empty.
-  const errorMessage = `${description}${
-    description && message ? ': ' : ''
-  }${message}`;
+  const message = formatErrorMessage(error, /** prefix */ description);
```

Comment:
```suggestion
  const message = formatErrorMessage(error, /* prefix */ description);
```

This isn't a JSDoc comment.